Extensions/Physics2Behavior/physics2runtimebehavior.ts

```diff
@@ -377,6 +379,9 @@ namespace gdjs {
     }
 
     onDeActivate() {
+      this.contactsEndedThisFrame.length = 0;
```

Comment:
Any risk doing this if we deactivate and then reactive the behavior during the same frame?

Reply:
I guess that the object holding the behavior will forget about its contacts, but not the other objects it was in contact with. So I would say it's not too risky

Reply:
Related question: should there be a `if (this._body !== null)` in onContactEnd?

Like, I'm deactivating a physics behavior of an object, so the body is deleted, so the contact with another objects stops.... so `EndContact` is called and the other object is added back to `contactsEndedThisFrame`, despite you just clearing this array before (and it will stay there until it's reactivated and doStepPreEvents is called).

Reply:
I was wondering about that at one point. I'll with the tests if this can be checked.